src/components/Switch/Switch.component.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SwitchContainer = styled.div`
```

Comment:
These styles should be declared in a separated file e.g `Switch.styled.js` to separate styling from the component.